src/plugins/eqht-css/reports/1-acr-wcag21.json

```diff
@@ -0,0 +1,483 @@
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/eqht-css/reports/1-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-05-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:focus-order",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/eqht-css/reports/1-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-05-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:link-purpose-in-context",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/eqht-css/reports/1-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-05-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:multiple-ways",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/eqht-css/reports/1-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-05-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:headings-and-labels",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/eqht-css/reports/1-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-05-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:focus-visible",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/eqht-css/reports/1-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-05-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG22:focus-appearance-minimum",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/eqht-css/reports/1-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-05-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    }
+  ]
+}
```

Comment:
Missing the following success criterion in the Conformance report:

* Success Criterion 2.5.1 Pointer Gestures
* Success Criterion 2.5.2 Pointer Cancellation
* Success Criterion 2.5.3 Label in Name
* Success Criterion 2.5.4 Motion Actuation
* Success Criterion 3.1.1 Language of Page
* Success Criterion 3.1.2 Language of Parts
* Success Criterion 3.2.1 On Focus
* Success Criterion 3.2.2 On Input
* Success Criterion 3.2.3 Consistent Navigation
* Success Criterion 3.2.4 Consistent Identification
* Success Criterion 3.3.1 Error Identification
* Success Criterion 3.3.2 Labels or Instructions
* Success Criterion 3.3.3 Error Suggestion
* Success Criterion 3.3.4 Error Prevention (Legal, Financial, Data)
* Success Criterion 4.1.1 Parsing
* Success Criterion 4.1.2 Name, Role, Value
* Success Criterion 4.1.3 Status Messages

You can copy/paste the missing JSON content from the ACR example here: https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/acr-wcag21-en.html

Reply:
Confirm this is completed